File: Qt_turtle_service/demo/ROS_demo/turtle_demo/src/turtle_param.cpp
#include "ros/ros.h"


int main(int argc, char *argv[])
{
    ros::init(argc,argv,"turtle_param");

    ros::NodeHandle nh("turtlesim");

    nh.setParam("background_r",0);
    nh.setParam("background_g",0);
    nh.setParam("background_b",0);

    //ros::NodeHandle nh;

    // ros::param::set("/turtlesim/background_r",0);
    // ros::param::set("/turtlesim/background_g",0);
    // ros::param::set("/turtlesim/background_b",0);
    
    return 0;
}

File: ROS_demo/turtle_demo/scripts/turtle_param.py
#! /usr/bin/env python
# coding:utf-8

import rospy

if __name__ == "__main__":
    node_name = "turtle_param"
    rospy.init_node(node_name)

    # rospy.set_param("/turtlesim/background_r",255)
    # rospy.set_param("/turtlesim/background_g",255)
    # rospy.set_param("/turtlesim/background_b",255)
    rospy.set_param("background_r",0)
    rospy.set_param("background_g",0)
    rospy.set_param("background_b",0)  # 调用时，在命令窗口需要传入arg: __ns:=/turtlesim